resources/aws_ec2.go

```diff
@@ -66,17 +87,31 @@ var AwsRegions = []string{
 // AWS credentials must be present in ~/.aws - For detailed instructions see
 // http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
 type AwsEc2Res struct {
-	BaseRes `yaml:",inline"`
-	State   string `yaml:"state"`   // state: running, stopped, terminated
-	Region  string `yaml:"region"`  // region must match an element of AwsRegions
-	Type    string `yaml:"type"`    // type of ec2 instance, ie. t2.micro
-	ImageID string `yaml:"imageid"` // imageid must be available on the chosen region
+	BaseRes     `yaml:",inline"`
+	State       string `yaml:"state"`       // state: running, stopped, terminated
+	Region      string `yaml:"region"`      // region must match an element of AwsRegions
+	Type        string `yaml:"type"`        // type of ec2 instance, ie. t2.micro
+	ImageID     string `yaml:"imageid"`     // imageid must be available on the chosen region
+	SnsEndpoint string `yaml:"snsendpoint"` // the public url endpoint to subscribe to events, ie. http://server:12345/
+	SnsPort     uint16 `yaml:"snsport"`     // the local port connected to the above url, ie. 23456
```

Comment:
I think you've simplified the API to only specify a listen port, why not allow the whole connection addr? Eg: why not `SnsListenAddr` or something. Of course it's fine if you find only an int that you turn it into fmt.Sprintf(":%d, theint), but otherwise maybe they only want to listen on the 24.12.2.5 network instead of 0.0.0.0 ?